packages/web-components/src/presence-badge/presence-badge.styles.ts

```diff
@@ -0,0 +1,107 @@
+import { css } from '@microsoft/fast-element';
+
+export const styles = css`
+  :host {
+    align-items: center;
+    box-sizing: border-box;
+    cursor: default;
+    display: inline-flex;
+    flex-direction: row;
+    font-size: 0;
+    line-height: 0;
+    min-height: 6px;
+    min-width: 6px;
+    position: relative;
+    transition: opacity 0.5s linear;
+    --presence-size: 16px;
+    --presence-mask-width: 2px;
+    --presence-mask-color: #eee;
```

Comment:
is the mask color meant to match the background color? Is the a color recipe we could use instead?

Reply:
A mask color isn't going to work considering the Windows 11 design language that works a lot on opacity. If this is on a mica surface the ring will show. I suggest we take this out of this component and work on a true masking cutout that can apply to a container. For instance, this component is primarily intended for use over an avatar, and _that's_ where the cutout actually needs to happen. This is true for rare usage over an icon as well. I started playing with this once in svg, but didn't get it into the system yet.

Reply:
Per my comment above, let's remove the mask now that it's being removed from usage in the design.